source/containers.tex

```diff
@@ -76,8 +76,8 @@
 \tcode{T}, \tcode{a} and \tcode{b}
 denote values of type \tcode{X}, \tcode{u}
```

Comment:
Should `b` be of type `X` or `const X`? If not, why do we have both `b`and `r` with different descriptions?